test/test_auth_service.py

```
@@ -0,0 +1,130 @@
+import pytest
+from src.application.services.auth_service import AuthService, DEFAULT_NICKNAME, DEFAULT_PROFILE_IMAGE
+from src.infrastructure.database.sqlalchemy_user_repository import SqlAlchemyUserRepository
+from .fixtures import KakaoUserInfoFixture, UserFixture
```

Ruff (0.12.2)

`.fixtures.UserFixture` imported but unused

Remove unused import: `.fixtures.UserFixture`

(F401)